docs/Highlight.md

```diff
@@ -6,115 +6,84 @@ It would be good to support multiple highlighter engines, but also allow for
 sane defaults. As such, two entry points will be available. One is a simplified
 syntax, and the other is a more complicated one.
 
-### Simple Syntax
+### Command Syntax
 
 ```
-FT.SEARCH ... SUMMARIZE
-    [TAGS {open} {close}]
-    [FRAGSIZE {size}]
-    [NOTRUNCATE]
-    {num} {field}
-```
-
-* **TAGS** If passed, should specify the open and close tags to surround each
-    term with.
-    If not specified, then no special text will wrap the terms.
-* **FRAGSIZE** return (approximately) {size}-sized summaries. If the raw text is
-    less than this size, or within +/- 10 chars, it is not truncated.
-    If not specified, the module default will be used.
-* **NOTRUNCATE** Rather than returning a synopsis of the field, return the entire
-    field's body with highlighting applied.
-
-* **num** number of fields to summarize
-* **field** fields to summarize
-
-The return value will be augmented to the fields returned via `RETURN`.
-Conversely, if `RETURN` appears after this keyword, then the fields indicated
-in `RETURN` follow after those from `SUMMARIZE`.
+FT.SEARCH ...
+    SUMMARIZE [FIELDS {num} {field}] [FRAGS {numFrags}] [LEN {fragLen}]
+    HIGHLIGHT [FIELDS {num} {field}] [TAGS {openTag} {closeTag}]
```

Comment:
what about separator?
also len should be the total len, not the frag len.

Reply:
The total len of what? if so, why allow the number of fragments? `LEN` indicates how much context each fragment should have, and `FRAGS` indicates the maximum number of samples of the matching term should be returned.

`LEN` cannot be used to limit the size of the block returned. In theory it's possible for a single fragment to be very very large; if a matching term exists repeatedly in the same field. This is actually something we should fix...